### ISSUE TITLE
Digest should show why something has been updated in the past week

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#2234]
https://publiclab.org/subscriptions/digest now shows (to logged-in people) a list of things updated in the past week that you subscribe to. But for many of them, it's not clear why they're in the list. It doesn't look like the authors of several of these actually updated their posts.

![screenshot 2018-02-05 at 11 28 27 am](https://user-images.githubusercontent.com/24359/35815973-b484d478-0a67-11e8-9a31-44d5048cd795.png)

On closer look, these are listed because a tag was added to them in the past week, a tag I'm following. 

Do we need to tweak this? We base this on this method:

https://github.com/publiclab/plots2/blob/fca26f2cb3935269f0135e7be14c6f7c708e8a41/app/models/user.rb#L313-L320

But this collects things that have been updated in that period -- does adding a tag to a node update it's `changed` timestamp? Maybe we should only show things that were created during that period, not all things that were updated? 

Keep in mind this list is going to be used to form a "digest email" as in our email notifications overhaul!

https://github.com/publiclab/plots2/milestone/9

**Comment by Gauravano** [comment]
Hi @jywarren, I can let you know if by any way we can skip those nodes with tag addition changes by running few tests.Otherwise, we can add only those nodes which are created in past week like you said which will be more definite and will highlight only new content. This will be awesome once completed with notification.Can I work on this?

**Comment by jywarren** [comment]
sure, that sounds great, and thanks!

**Comment by Gauravano** [comment]
Hi @jywarren after running some tests, I have solved the problem of inclusion of wiki in digest on just tag addition.It's ready, but I think if we want to send the digest to a user via email we should include new wikis only. I have both branches ready for edited page only and newly created wiki.Please suggest with which should I raise PR. Thanks.

**Comment by Gauravano** [comment]
hey, @jywarren plz provide your input here.thanks.

**Comment by jywarren** [comment]
Yes, i think you're right and we can just skip the tag additions. They were originally added because we would immediately send an email, and if the tag wasn't added at the same time as the node published, no email would be sent. But for these digests, there'll be a window of probably 8 hours minimum -- and often days, if it's weekly -- for the tags to be added. So the extra notification pegged to tag creation is not needed. 

Great! Thanks!
